test-integration/helpers/reading-builder.coffee

```diff
@@ -269,10 +273,11 @@ class ReadingBuilder extends TestHelper
           @cancel() if isDisplayed
       else
         # Click "Add Readings"
+        @test.sleep(100) # Not sure why this is needed
         @el.addReadingsButton.click()
```

Comment:
My guess for why some of the timeouts like this are needed is because the `async-button` will very briefly in the 'waiting' state.  If selenium's `click` is called at that instant then it's ignored.